packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -0,0 +1,177 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-map-marker');
+const { generateTemplate } = storybookTemplate('sd-map-marker');
+
+export default {
+  title: 'Components/sd-map-marker',
+  component: 'sd-map-marker',
+  parameters: {
+    ...parameters
+  },
+  argTypes,
+  decorators: [withActions] as any
+};
+
+/**
+ * This shows the badge in its default state and variant.
+ */
+
+export const Default = {
+  parameters: { controls: { exclude: ['default'] } },
```

Comment:
I am not sure I understand this line @saemik94 . What are you trying to exclude?

Reply:
yes I exclude the default, because the slot only makes sense to use for the 'cluster' and 'place'. Hope that makes sense.